"px68k_midi_output" and "px68k_midi_output_type" invalid value errors

Getting this in logs every time the core boots:

> [ERROR] [Environ]: GET_VARIABLE: px68k_midi_output - Invalid value.
[ERROR] [Environ]: GET_VARIABLE: px68k_midi_output_type - Invalid value.

This is because this function still exists: https://github.com/libretro/px68k-libretro/blob/4c7542e34df71e3ebab77b5867901747d58dbb38/libretro.c#L320-L349

while the core options were removed in commit 9ac92513a16c90df3ef2eed39572a24fbc4d75f2

I wanted to make a PR, but idk if I was supposed to remove the function or re-add the 2 core options (I have no idea if they work, I never messed with the MIDI stuff) :p So I prefer to open an issue about it instead.

--- a/libretro_core_options.h
+++ b/libretro_core_options.h
@@ -156,6 +156,30 @@ struct retro_core_option_definition option_defs_us[] = {
       "Default"
    },
    {
+      "px68k_midi_output",
+      "MIDI Output (Restart)",
+      "Enable software MIDI.",
+      {
+         { "disabled", NULL},
+         { "enabled",  NULL},
+         { NULL,       NULL },
+      },
+      "enabled"
+   },
+   {
+      "px68k_midi_output_type",
+      "MIDI Output Type (Restart)",
+      "Sets MIDI output type.",
+      {
+         { "LA",       NULL },
+         { "GM",       NULL },
+         { "GS",       NULL },
+         { "XG",       NULL },
+         { NULL,       NULL },
+      },
+      "GM"
+   },
+   {
       "px68k_adpcm_vol",
       "ADPCM Volume",
       "Sets the volume of the ADPCM sound channel.",

